.appveyor.yml

```diff
@@ -58,16 +58,18 @@ matrix:
 install:
   - ps: >-
       $env:CONFIGOPTS="--disable-documentation"
+  - ps: >-
+      $env:BINDINGS=""
   - ps: >-
       if ($env:TOOLCHAIN -eq "mingw") {
         $env:PATH="C:\Mingw\bin;C:\Mingw\MSYS\1.0\bin;$env:PATH"
       } elseif ($env:TOOLCHAIN -eq "mingw64") {
         if ($env:Platform -eq "x86") {
           $env:PATH="C:\mingw-w64\i686-6.3.0-posix-dwarf-rt_v5-rev1\mingw32\bin;C:\msys64\usr\bin;$env:PATH"
-          $env:CONFIGOPTS+=" --host=i686-pc-mingw32"
         } else {
           $env:PATH="C:\mingw-w64\x86_64-7.2.0-posix-seh-rt_v5-rev1\mingw64\bin;C:\msys64\usr\bin;$env:PATH"
-          $env:CONFIGOPTS+=" --host=x86_64-w64-mingw32"
+          $env:CONFIGOPTS+=" --host=x86_64-w64-mingw32 --with-go"
+          $env:BINDINGS+=" xapian-bindings "
```

Comment:
I'd just set (rather than append) since `BINDINGS` is either empty or `xapian-bindings`.  And I'd drop the extra spaces around the value - they aren't needed and make the substituted command-line look a bit messy.